llvm/utils/TableGen/InstrInfoEmitter.cpp

```diff
@@ -24,7 +24,6 @@
 #include "llvm/ADT/STLExtras.h"
 #include "llvm/ADT/SmallVector.h"
 #include "llvm/ADT/StringExtras.h"
-#include "llvm/Support/Casting.h"
```

Comment:
This file does use `cast` etc.  The header is rarely included directly, but if we follow the IWYU principle, it should probably be included more often.